dss/dynamodb/__init__.py

```diff
@@ -3,52 +3,83 @@
 from dss.util.aws.clients import dynamodb as db  # type: ignore
 
 
-def put_item(table: str, value: str, hash_key: str, sort_key: str):
+def put_item(table: str, value: str, hash_key: str, sort_key: str=None):
     """
     Put an item into a dynamoDB table.
 
+    Will determine the type of db this is being called on by the number of keys provided (omit
+    sort_key to PUT into a db with only 1 primary key).
+
     :param table: Name of the table in AWS.
     :param str value: Value stored by the two primary keys.
     :param str hash_key: 1st primary key that can be used to fetch associated sort_keys and values.
     :param str sort_key: 2nd primary key, used with hash_key to fetch a specific value.
+                         Note: If not specified, this will PUT only 1 key (hash_key) and 1 value.
     :return: None
     """
-    db.put_item(
-        TableName=table,
-        Item={
-            'hash_key': {
-                'S': hash_key
-            },
-            'sort_key': {
-                'S': sort_key
-            },
-            'body': {
-                'S': value
-            },
-        }
-    )
+    if sort_key:
+        db.put_item(
+            TableName=table,
+            Item={
+                'hash_key': {
+                    'S': hash_key
+                },
+                'sort_key': {
+                    'S': sort_key
+                },
+                'body': {
+                    'S': value
+                },
+            }
+        )
+    else:
+        db.put_item(
+            TableName=table,
+            Item={
+                'key': {
+                    'S': hash_key
+                },
+                'body': {
+                    'S': value
+                },
+            }
+        )
```

Comment:
if the async ddb schema is changed to use `hash_key` instead of `key`, this will work:
```
item = {
    'hash_key': {
        'S': hash_key
    },
    'body': {
        'S': value
    }
if sort_key is not None:
    item['sort_key'] = dict(S=sort_key)
db.put_item(TableName=table, Item=item)
```

Reply:
Alright, can do.  What additional steps should I take when tearing down the table and building the new one?